app/pages/project/networking/VpcPage/VpcPage.tsx

```diff
@@ -50,15 +50,15 @@ export function VpcPage() {
         </PropertiesTable>
         <PropertiesTable>
           <PropertiesTable.Row label="Creation Date">
-            {vpc.timeCreated && formatDateTime(vpc.timeCreated)}
+            {formatDateTime(vpc.timeCreated)}
           </PropertiesTable.Row>
           <PropertiesTable.Row label="Last Modified">
-            {vpc.timeModified && formatDateTime(vpc.timeModified)}
+            {formatDateTime(vpc.timeModified)}
```

Comment:
checks were holdovers from before this was prefetched and therefore never undefined